test/Spec.hs

```diff
@@ -39,24 +36,31 @@ import Text.Read (readEither)
 
 main :: IO ()
 main = hspec $ do
-  lexerSpec
+  --lexerSpec
   parserSpec
   evalSpec
 
 --------------------------------------------------------------------------------
 -- Lexing tests.
 
 -- | Lexer tests.
+-- TODO: Round Trip tests don't make sense with spans unless we also
+-- have a pretty printer. This test should be reintroduced and
+-- rewritten once we have a working pretty printer.
 lexerSpec :: Spec
 lexerSpec = describe "Lexer" $
   describe "QuickCheck" $
-    it "lexing serialized tokens yields those tokens" $
+    -- Note: This should be a pretty printer round trip test to account for spans
+    it "lexing serialized tokens yields those tokens modulo spans" $
```

Comment:
```suggestion
    it "lexes serialized tokens and yields those tokens modulo spans" $
```
`hspec` tests should read almost like sentences (e.g. "it lexes serialized tokens and yields those tokens modulo spans").